docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,724 @@
+        "organization": "Org1",
+        "connection": {
+            "timeout": {
+                "peer": {
+                    "endorser": "300"
+                }
+            }
+        }
+    },
+    "organizations": {
+        "Org1": {
+            "mspid": "Org1MSP",
+            "peers": [
+                "peer0.org1.example.com"
+            ],
+            "certificateAuthorities": [
+                "ca.org1.example.com"
+            ]
+        }
+    },
+    "peers": {
+        "peer0.org1.example.com": {
+            "url": "grpcs://localhost:7051",
+            "tlsCACerts": {
+                "pem": "-----BEGIN CERTIFICATE-----<UNIQUE CONTENT>-----END CERTIFICATE-----\n"
+            },
+            "grpcOptions": {
+                "ssl-target-name-override": "peer0.org1.example.com",
+                "hostnameOverride": "peer0.org1.example.com"
+            }
+        }
+    },
+    "certificateAuthorities": {
+        "ca.org1.example.com": {
+            "url": "https://localhost:7054",
+            "caName": "ca-org1",
+            "tlsCACerts": {
+                "pem": "-----BEGIN CERTIFICATE-----<UNIQUE CONTENT>-----END CERTIFICATE-----\n"
+            },
+            "httpOptions": {
+                "verify": false
+            }
+        }
+    }
+}
+```
+### Version
+Ensure that version is set to `"1.0"` as Caliper uses a validator on the configuration file that only accepts the "1.0" option.
+
+### Caliper Object
+
+Identify the Distributed Ledger Technology engine that is being tested. Add an object called `caliper` to the schema that contains the property `blockchain` with the string value `fabric`. 
+
+It should look like this:  
+
+```bash 
+"caliper": {
+        "blockchain": "fabric"
+    },
+
+```
+
+### Clients Object
+
+Specify the identities to be used by Caliper when performing the benchmark. Identities *must* be valid, meaning they must be known to the Fabric network and have the corresponding crypto material for use. Identities are listed within a `clients` stanza. Add an object called `clients` to the schema and a property named after one of the identities in the network. In this case the identity used is `Admin@org1.example.com`. Then nest the existing `client` object from the CCP within the identity. 
+
+``` bash
+"clients": {
+    "Admin@org1.example.com": {
+        "client": {
+            "connection": {
+                "timeout": {
+                    "peer": {
+                        "endorser": "300"
+                    }
+                }
+            }
+        }
+    }
+},
+```
+
+Under the `client` object add a property called `credentialStore` under this property add a property called `path` that has a string variable leading to a temp file in the workspace with the name `tmp/hfc-kvs/org1`. Also under the `credentialStore` property add a property called `cryptoStore` and under this add another `path` property that points to the same temp file above, `tmp/hfc-kvs/org1`. 
```

Comment:
Need a comma or fullstop before the 2nd `under` in this statement
The tmp/hfc-kvs/org1, I think are directories rather than temp files. Also should we specify /tmp rather than tmp ?

Reply:
So far I personally specified /tmp, and it was cleaned up with the end script. It could also point to the workplace-relative tmp dir, but it also needs to be cleaned up when the crypto artifacts change.